contrib/src/json/mod.rs

```diff
@@ -129,20 +129,23 @@ impl<T> DerefMut for JSON<T> {
 /// ```
 ///
 /// The return type of a macro invocation is
-/// [Value](/rocket_contrib/enum.Value.html). A value created with this macro
+/// [Value](/rocket_contrib/enum.Value.html), and is the default type parameter 
+/// of the JSON type. A value created with this macro
```

Comment:
Also, I'd like a slightly longer explanation. Kill the `,` and add another sentence explaining that it's the default value and as such you can return `JSON` without specifying the type.